src/glyph/Glyph.js

```diff
@@ -29,7 +29,7 @@ export default class Glyph {
     this._font = font;
 
     // TODO: get this info from GDEF if available
-    this.isMark = this.codePoints.every(unicode.isMark);
+    this.isMark = this.codePoints.length > 0 && this.codePoints.every(unicode.isMark);
```

Comment:
possible alternative, `[].reduce((isMark,glyph) => isMark && glyph.isMark, false)`?

Reply:
The version here is really incorrect as it should be getting the info from the GDEF table as GlyphInfo does. I've been meaning to just remove it from Glyph since it is unused - all opentype processing uses GlyphInfo.